### ISSUE TITLE
Setting `EMQ_ADMIN_PASSWORD` fails

### ISSUE THREAD

**Issue by a3aanwisse** [emqtt/emq-docker#42]
### Expected behavior
When specifying EMQ_ADMIN_PASSWORD=testpassword it is being used as specified. However the default password is still used (being 'public')

### Actual behavior
The following appears in the logfile

`emqtt_1 | ['2018-01-29T12:55:17Z']:admin password changed to testpassword
emqtt_1 | Usage: emqttd_ctl
emqtt_1 | --------------------------------------------------------------------------------
emqtt_1 | status                                          # Show broker status
emqtt_1 | --------------------------------------------------------------------------------
emqtt_1 | broker                                          # Show broker version, uptime and description
emqtt_1 | broker pubsub                                   # Show process_info of pubsub
emqtt_1 | broker stats                                    # Show broker statistics of clients, topics, subscribers
emqtt_1 | broker metrics                                  # Show broker metrics
emqtt_1 | --------------------------------------------------------------------------------
emqtt_1 | cluster join <Node>                             # Join the cluster
emqtt_1 | cluster leave                                   # Leave the cluster
emqtt_1 | cluster force-leave <Node>                      # Force the node leave from cluster
emqtt_1 | cluster status                                  # Cluster status
emqtt_1 | --------------------------------------------------------------------------------
emqtt_1 | users list                                      # List users
....
`
### Test code

docker-compose:
```version: '2'
services:
  emqtt:
    build: .
    restart: "always"
    environment:
      - EMQ_LOADED_PLUGINS="emq_dashboard,emq_auth_http,emq_modules,emq_retainer"
      - EMQ_ADMIN_PASSWORD=testpassword
    ports:
      - "1883:1883"
      - "8883:8883"
      - "18083:18083" 
```

### EMQ version
2.3.4

### Docker version
Docker version 17.12.0-ce, build c97c6d6

**Comment by a3aanwisse** [comment]
When adding emq_recon to the list with EMQ_LOADED_PLUGINS it succeeds...

**Comment by 7wonders** [comment]
With emq_recon and emq_dashboard it actually still fails quite often, as though the CLI has not had time to initiate. To prove this I added a 5 sec sleep which seems to work well as a workaround:

```
if [[ ! -z "$EMQ_ADMIN_PASSWORD" ]]; then
    sleep 5
    echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:admin password changed to $EMQ_ADMIN_PASSWORD"
    /opt/emqttd/bin/emqttd_ctl admins passwd admin $EMQ_ADMIN_PASSWORD &
fi
```

**Comment by emqplus** [comment]
@7wonders @a3aanwisse I submit a PR #43 to fix the issue. Please help review it.

**Comment by emqplus** [comment]
@7wonders @a3aanwisse Fixed in 2.3.5
